app/views/two_factor_authentication/otp_verification/show.html.slim

```diff
@@ -15,4 +15,4 @@ h1.h3.my0 = @presenter.header
   = submit_tag t('forms.buttons.submit.default'), class: 'btn btn-primary align-top'
 
 = render 'shared/fallback_links', presenter: @presenter
-= render 'shared/cancel', link: @presenter.cancel_link
+= render 'shared/cancel', link: @presenter.cancel_link, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.